src/database/XML/xml.sql

```diff
@@ -1,7 +1,14 @@
-SELECT xmlelement(
-	name value, 
-	xmlattributes(comsumption_cod as comsumption),
-	xmlelement( name nif, nif),
-	xmlelement( name comsumption_date, comsumption_date)
-)
-FROM Comsumptions
\ No newline at end of file
+CREATE OR REPLACE PROCEDURE XML_Comsumptions()
```

Comment:
This should be a function since it is a requirement to return a XML.
The purpose of it is, essentially, display the XML data do the client (even though it might not be implemented).